core/src/main/scala/org/apache/spark/network/netty/SparkTransportConf.scala

```diff
@@ -43,8 +43,12 @@ object SparkTransportConf {
    * @param numUsableCores if nonzero, this will restrict the server and client threads to only
    *                       use the given number of cores, rather than all of the machine's cores.
    *                       This restriction will only occur if these properties are not already set.
+   * @param disablePortRetry if true, server will not retry its port. It's better for the long-run
+   *                        server to disable it since the server and client had the agreement of
+   *                        the specific port.
```

Comment:
actually, since the only place we ever set this to true is in `StandaloneWorkerShuffleService`, does it make sense to just move the conf setting there along with this comment? (similar to what you did in YARN)